examples/mysql-wordpress-pd/README.md

```diff
@@ -34,301 +34,246 @@ Documentation for other releases can be found at
-
-We will create two disks: one for the mysql pod, and one for the wordpress pod. In this example, we create 20GB disks, which will be sufficient for this demo. Feel free to change the size to align with your needs, as wordpress requirements can vary. Also, keep in mind that [disk performance scales with size](https://cloud.google.com/compute/docs/disks/#comparison_of_disk_types).
-
-First create the mysql disk.
-
-```sh
-gcloud compute disks create --size=20GB --zone=$ZONE mysql-disk
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+## Table of Contents
+
+  - [Cluster Requirements](#cluster-requirements)
+  - [Decide where you will store your data](#decide-where-you-will-store-your-data)
+    - [GCE Persistent Disk](#gce-persistent-disk)
+    - [Host Path](#host-path)
+  - [Create the MySQL Password Secret](#create-the-mysql-password-secret)
+  - [Deploy MySQL](#deploy-mysql)
+  - [Deploy WordPress](#deploy-wordpress)
+  - [Visit your new WordPress blog](#visit-your-new-wordpress-blog)
+  - [Take down and restart your blog](#take-down-and-restart-your-blog)
+  - [Next Steps](#next-steps)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Cluster Requirements
+
+Kubernetes runs in a variety of environments and is inherently
+modular. Not all clusters are the same. These are the requirements for
+this example.
+
+* Kubernetes version 1.2 is required due to using newer features, such
+  at PV Claims and Deployments.
+* [Cluster DNS](../../cluster/addons/dns/) will be used for service discovery.
+* An [external load balancer](https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/user-guide/services.md#type-loadbalancer)
+  will be used to access WordPress.
+* [Persistent Volume Claims](https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/user-guide/persistent-volumes.md)
+  are used. You must create Persistent Volumes in your cluster to be
+  claimed. This example demonstrates how to create two types of
+  volumes, but any volume is sufficient.
+
+Consult a
+[Getting Started Guide](https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/getting-started-guides/README.md)
+to set up a cluster and the
+[kubectl](https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/user-guide/prereqs.md) command line client.
+
+## Decide where you will store your data
+
+Both MySQL and WordPress will have a
+[Persistent Volume](https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/user-guide/persistent-volumes.md) to
+store their data. We will use a Persistent Volume Claim to claim an
+available persistent volume. This example covers HostPath and
```

Comment:
HostPath isn't used with Persistent Volumes, so this is confusing. Rephrase so that HostPath and Persistent Volume are 2 distinct cases.